AgentQnA/tests/run_test_agent_service.sh

```diff
@@ -0,0 +1,8 @@
+# Copyright (C) 2024 Intel Corporation
```

Comment:
This script won't be auto triggered.

Reply:
@chensuyue Correct, this is not part of the test. I can remove it.